signing/softhsm/softhsm.conf

```diff
@@ -0,0 +1,15 @@
+# SoftHSM v2 configuration file
+directories.tokendir = signing/softhsm/tokens
+objectstore.backend = file
+
+# ERROR, WARNING, INFO, DEBUG
+log.level = WARNING
+
+# If CKF_REMOVABLE_DEVICE flag should be set
+slots.removable = false
+
+# Enable and disable PKCS#11 mechanisms using slots.mechanisms.
+slots.mechanisms = ALL
+
+# If the library should reset the state on fork
+library.reset_on_fork = true
```

Comment:
The manpage isn't very forthcoming about why you'd use this, but it seemed sensible to me.  Internally, it discards and recreates the token's state, which would include closing all open file descriptors owned by the library (which was my main motivation for setting it).

It may be a bit of a non-sequitur though: I don't think any of our use cases actually `fork`.